cli/lib/api.js

```diff
@@ -1,21 +1,112 @@
 const fetch = require('node-fetch');
+const zlib = require('zlib');
 
 class Api {
-  constructor(baseUrl) {
+  constructor(baseUrl, token) {
+    this.token = token;
     this.baseUrl = baseUrl;
+    this.baseUrlV1 = `${baseUrl}/v1`;
   }
 
-  headers(token) {
+  headers() {
     return {
-      authorization: `Bearer ${token}`
+      authorization: `Bearer ${this.token}`
```

Comment:
I decided to inherit this from `main.js` since a user _should_ be logged in at runtime for a command that needs to be auth'd